test/integration/test_bind.rb

```diff
@@ -10,7 +10,9 @@ def test_bind_timeout
     error = assert_raise Net::LDAP::Error do
       @ldap.bind(method: :simple, username: "uid=user1,ou=People,dc=rubyldap,dc=com", password: "passworD1")
     end
-    assert_equal('Connection timed out - user specified timeout', error.message)
+    msgs = ['Operation timed out - user specified timeout',
+            'Connection timed out - user specified timeout']
+    assert_send([msgs, :include?, error.message])
```

Comment:
This is the tcp/8389 timeout check that requires the iptables `OUTPUT`/`INPUT` change. They are actually the same error. OpenSSL generates the error message text, and it provides different text on macOS:
```bash
tmaher@turing:~$ openssl s_client -connect 172.28.128.3:8389
connect: Operation timed out
connect:errno=60
```
...versus on Linux in the VM...
```
vagrant@rubyldap:~$ openssl s_client -connect 172.28.128.3:8389
connect: Connection timed out
connect:errno=110
```
This is why further down int his file I'm also rescuing two different possible exceptions for the same test: macOS vs Linux have different results. Would it make more sense for `#bind` and other bits to normalize the timeout error message, so it's the same across all platforms?